lsp/src/lib.rs

```diff
@@ -129,11 +130,7 @@ pub fn start_lsp() -> Result<()> {
         text_document_sync: Some(TextDocumentSyncCapability::Kind(TextDocumentSyncKind::FULL)),
         completion_provider: Some(lsp_types::CompletionOptions {
             resolve_provider: Some(false),
-            trigger_characters: Some(vec![
-                "-".to_string(),
-                "\"".to_string(),
-                " ".to_string(),
-            ]),
+            trigger_characters: Some(vec!["-".to_string(), "\"".to_string(), " ".to_string()]),
```

Comment:
hmm we may have a rustfmt issue here... maybe?

probably should just have one in the project